### ISSUE TITLE
Home connect - Bosch

### ISSUE THREAD

**Issue by renatoyamane** [hagezi/dns-blocklists#3087]
### Which AdBlocker/DNS cloud service do you use?

AdGuard DNS, ControlD, NextDNS

### ControlD users

- [X] IMPORTANT - I assure that I have not set the Block Response in ControlD to Custom or Branded and can confirm that the problem still occurs.

### NextDNS users

- [X] IMPORTANT - I can assure that I disabled the block page in NextDNS and can confirm that the problem still occurs.

### With which block list(s) does the problem occur?

Multi PRO++

### Which domain(s) should be unblocked?

bshg.data.adobedc.net

### Why should the domain(s) be unblocked?

It's blocking the access to my Bosch Dishwasher when using the Home Connect app:
https://www.home-connect.com/global/works-with/smart-home/bosch-smart-home

**Comment by hagezi** [comment]
@renatoyamane I also use the app and have no problems with the connection to the dishwasher. However, I also don't see any call in the log to this Adobe tracker. I use the iOS app. What steps are necessary to trigger this?

**Comment by hagezi** [comment]
@xRuffKez Didn't you also use Home Connect?

**Comment by xRuffKez** [comment]
> @xRuffKez Didn't you also use Home Connect?

Yes, what's the matter?

**Comment by xRuffKez** [comment]
![Screenshot_20240707_223253_Home Connect.jpg](https://github.com/hagezi/dns-blocklists/assets/92758042/7706ca48-6e08-48a6-9186-7c423a493e79)

![Screenshot_20240707_223311_Home Connect.jpg](https://github.com/hagezi/dns-blocklists/assets/92758042/8e53c691-9295-404b-97d8-332ee3b254c9)

No issues, Domain is not called once.

**Comment by hagezi** [comment]
Thanks, then let's wait for @renatoyamane, maybe we're missing something. The domain has also been blocked for ages and there are no reports about it.

**Comment by renatoyamane** [comment]
When my phone is connected to the WiFi, I can't connect to the dishwasher.

DNS servers on the router is set to use the Pro++ filter.

Turning the WiFi off and using only the mobile data, my phone can connect to the Dishwasher normally.

When using the WiFi:
![Screenshot_20240707-213100~2](https://github.com/hagezi/dns-blocklists/assets/31208391/603355f3-c923-43a7-8c1e-db4d40b76331)

**Comment by xRuffKez** [comment]
I played with the App. If i go to "Shop" and click on spare parts, a pdf opens and the domain is called. Blocked but everything works in the App

**Comment by hagezi** [comment]
@renatoyamane And if you allow the domain, does it also work with WiFI? I can't really imagine that.

**Comment by renatoyamane** [comment]
I'm using the DNS 
x-hagezi-proplus.freedns.controld.com

So not sure how to whitelist a domain on it
I'll do more checks tomorrow.

**Comment by hagezi** [comment]
With ControlD free you have no possibility to whitelist anything or view the logs, so what makes you think that your problem is caused by blocking bshg.data.adobedc.net?

I have now also tested again with ControlD free, Pro++ and Ultimate. No problems with the connection.

**Comment by renatoyamane** [comment]
I checked the logs on AdGuard Home, because it's the only thing blocked when I access the Home Connect:
![image](https://github.com/hagezi/dns-blocklists/assets/31208391/afa42431-5ac4-46ab-8e98-0cd54c70ba0a)

So I disabled it the problem was still present.
I assumed the problem was coming from ControID (HaGeZi Pro++), because it's the only list under use on my router when the AdGuard Home is disabled.

But because no one else is able to reproduce this problem, looks like the problem is in somewhere else on my side.
Sorry for wasting your time on this.
